apm-server.reference.yml

```diff
@@ -21,17 +21,28 @@ apm-server:
   #ssl.certificate : "path/to/cert"
   #ssl.key : "path/to/private_key"
 
-  #frontend.enabled: false
-
-  # Rate Limit per second and IP address
-  #frontend.rate_limit: 10
+  # In case frontend is enabled
+  # and the index pattern at 'outputs.elasticsearch.index' is customized,
+  # a matching 'frontend.sourcemapping.index' pattern needs to be specified
+  # to allow for sourcemapping to work properly. 
+  #frontend:
+    #enabled: false
 
-  # Comma separated list of permitted origins for frontend. User-agents will send
-  # a origin header that will be validated against this list.
-  # An origin is made of a protocol scheme, host and port, without the url path.
-  # Allowed origins in this setting can have * to match anything (eg.: http://*.example.com)
-  # If an item in the list is a single *, everything will be allowed
-  #frontend.allow_origins : *
+    # Rate Limit per second and IP address
+    #rate_limit: 10
+
+    # Comma separated list of permitted origins for frontend. User-agents will send
+    # a origin header that will be validated against this list.
+    # An origin is made of a protocol scheme, host and port, without the url path.
+    # Allowed origins in this setting can have * to match anything (eg.: http://*.example.com)
+    # If an item in the list is a single *, everything will be allowed
+    #allow_origins : *
+
+    #sourcemapping:
+      #cache:
+        #expiration: 300 #seconds
+        #cleanup_interval: 600 #seconds
+      #index: "apm-"
```

Comment:
It took me a while to understand that this is the prefix we use when searching for sourcemaps and that an asterisk gets added automatically. It's unrelated to where we store them when it gets uploaded. I think it would be good to have a line or two describing this some more. By the way, it looks like the default value is `apm`, not `apm-`.

Reply:
There is a description right above `frontend`, but I'll move it to be directly above the `index` field and try to make it more descriptive.